Source/WebKit/UIProcess/ProcessAssertion.h

```diff
@@ -47,8 +47,11 @@ OBJC_CLASS WKRBSAssertionDelegate;
 #endif // USE(RUNNINGBOARD)
 
 #if USE(EXTENSIONKIT)
-OBJC_CLASS _SEExtensionProcess;
-OBJC_PROTOCOL(_SEGrant);
+OBJC_CLASS BEWebContentProcess;
+OBJC_CLASS BENetworkingProcess;
+OBJC_CLASS BERenderingProcess;
+OBJC_PROTOCOL(BEProcessCapabilityGrant);
+using ExtensionProcess = std::variant<RetainPtr<BEWebContentProcess>, RetainPtr<BENetworkingProcess>, RetainPtr<BERenderingProcess>>;
```

Comment:
Note that given all the cases where you need to switchOn() this, it might look nicer to have a class to wrap this std::variant<> and abstract away all this complexity. e.g. this class would have an "invalidate()" member function which would do the `WTF::switchOn()` internally. Call sites would look a lot nicer I think.